app/models/incomeSourceDetails/TaxYear.scala

```diff
@@ -60,6 +61,10 @@ case class TaxYear(startYear: Int, endYear: Int) {
     LocalDate.of(this.endYear, April, Fifth)
   }
 
+  def toApiFormat: String = {
```

Comment:
you can name this method as "shortenTaxYear" or something similar to improve (semantic) readability of our code base

Reply:
renamed to .shortenTaxYearEnd